src/libraries/tests.proj

```diff
@@ -759,6 +759,12 @@
       AdditionalProperties="%(AdditionalProperties);_IsPublishing=true;Configuration=Release;PublishDir=$(XUnitLogCheckerLibrariesOutDir);ROOTFS_DIR=$(ROOTFS_DIR);RuntimeIdentifier=$(OutputRID)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(ArchiveTests)' == 'true' and '$(TargetOS)' == 'android' and '$(RuntimeFlavor)' == 'CoreCLR'">
+    <ProjectReference Remove="@(ProjectReference)" />
```

Comment:
I suppose this is needed since we remove the `'$(RuntimeFlavor)' == 'Mono'"` condition above, if we kept the condition above, could we just include the `Android.Device_Emulator.JIT.Test.csproj` here?

Reply:
The tests are included by default in:
```
<ProjectReference Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj"
                      Exclude="@(ProjectExclusions)"
                      Condition="'$(TestAssemblies)' == 'true'"
                      BuildInParallel="$(BuildTestInParallel)" />
```
To exclude the default set of tests, we set TestAssemblies to false.